data/3_deliveryMethodMeta.csv

```diff
@@ -79,18 +79,6 @@ UK_GUARANTEED_ROYAL_MAIL_SPECIAL_DELIVERY_1PM_SATURDAY_2500,1001,2500,2500,"Spec
 UK_GUARANTEED_ROYAL_MAIL_SPECIAL_DELIVERY_9AM_SATURDAY_50,0,50,50,"Special Delivery: Guaranteed by 9am Saturday"
```

Comment:
Looks like these were added by me when I merged in the new methods. I've removed them now.